docs/source/user/installing.rst

```diff
@@ -17,60 +17,192 @@ Our supported platforms are:
+
+You do not need to install the CUDA SDK from NVIDIA.
+
+
+Installing using pip on x86/x86_64 Platforms
+--------------------------------------------
+
+Binary wheels for Windows, Mac, and Linux are also available from `PyPI
+<https://pypi.org/project/numba/>`_.  You can install Numba using ``pip``::
+
+    $ pip install numba
+
+This will download all of the needed dependencies as well.  You do not need to
+have *LLVM* installed to use Numba (in fact, Numba will ignore all LLVM
+versions installed on the system) as the required components are bundled into
+the llvmlite wheel.
+
+To use CUDA with Numba installed by `pip`, you need to install the `CUDA SDK
+<https://developer.nvidia.com/cuda-downloads>`_ from NVIDIA.  Then you may need to
+set the following environment variables so Numba can locate the required
+libraries:
+
+  * ``NUMBAPRO_CUDA_DRIVER`` - Path to the CUDA driver shared library file
+  * ``NUMBAPRO_NVVM`` - Path to the CUDA libNVVM shared library file
+  * ``NUMBAPRO_LIBDEVICE`` - Path to the CUDA libNVVM libdevice *directory* which contains .bc files
+
+
+Enabling AMD ROCm GPU Support
+-----------------------------
+
+The `ROCm Platform <https://rocm.github.io/>`_ allows GPU computing with AMD
+GPUs on Linux.  To enable ROCm support in Numba,  conda is required, so begin
+with an Anaconda or Miniconda installation with Numba 0.40 or later installed.
+Then:
+
+1. Follow the `ROCm installation instructions <https://rocm.github.io/install.html>`_.
+2. Install ``roctools`` conda package from the ``numba`` channel::
+
+    $ conda install -c numba roctools
+
+See the `roc-examples <https://github.com/numba/roc-examples>`_ repository for
+sample notebooks.
+
+
+Installing on Linux ARMv7 Platforms
+-----------------------------------
+
+`Berryconda <https://https://github.com/jjhelmus/berryconda>`_ is a
+conda-based Python distribution for the Raspberry Pi.  We are now uploading
+packages to the ``numba`` channel on Anaconda Cloud for ARMv7-based boards,
+which currently incudes the the Raspberry Pi 2 and 3, but not the Pi 1 or
+Zero.  These can be installed using conda from the `numba` channel::
+
+    $ conda install -c numba numba
+
+Berryconda and Numba may work on other Linux-based ARMv7 systems, but this has
+not been tested.
+
+
+Installing on Linux ARMv8 (AArch64) Platforms
+---------------------------------------------
+
+Although many ARMv8 platforms are likely to require building Numba from
+source, we do build and test conda packages on the `NVIDIA Jetson TX2
+<https://www.nvidia.com/en-us/autonomous-machines/embedded-systems-dev-kits-modules/>`_.
+These packages rely on an infrequently updated port of Berryconda to ARMv8
+called `Jetconda <https://github.com/seibert/jetconda/releases>`_.  Jetconda
+is built on Ubuntu 16.04 running on the Jetson TX2, although the packages may
+work on other ARMv8 systems running Ubuntu 16.04.  Once Jetconda is installed,
+Numba can be installed from the ``numba`` channel
```

Comment:
Double colon `::` after `channel` else the conda command won't end up in a code block.